index.js

```diff
@@ -25,15 +53,30 @@ app.get('/', function(request, response) {
   response.render('index');
 });
 
-app.get('/api/entries', api.get);
+app.get('/api/entries', function (req, res){
+	api.get(req, res, EntryModel)
+});
+
+app.post('/api/entry', function (req, res){
+	api.post(req, res, EntryModel)
+});
+
+app.post('/api/entries', function (req, res){
+	api.multiPost(req, res, EntryModel)
+});
 
-app.post('/api/entries', api.post);
 
-app.get('/api/entries/:id', api.getById);
+app.get('/api/entries/:id', function (req, res){
+	 api.getById(req, res, EntryModel)
+});
 
-app.put('/api/entries/:id', api.put);
+app.put('/api/entries/:id', function (req, res){
+	api.put(req, res, EntryModel)
```

Comment:
Wouldn't a PUT API call be synonymous with an 'update'? Want to support this functionality?